README.md

```diff
@@ -1,6 +1,6 @@
 # ActiveJob::Cancel
 
-`activejob-cancel` provides cancel method to Active Job. Currently it supports only `Sidekiq`, `Delayed Job` and `resque`.
+`activejob-cancel` provides cancel method to Active Job. Currently it supports only `Sidekiq`, `Delayed Job`, `resque` and the ActiveJob `TestAdapter`.
```

Comment:
`Active Job` is correct component name ;)

Reply:
:+1: